templates/cloud-config-secondary.yaml

```diff
@@ -40,6 +40,11 @@ write_files:
   permissions: "0400"
   content: "${proxy_url}"
 
+- path: /etc/ptfe/custom-ca-cert-url
+  owner: root:root
+  permissions: "0400"
+  content: "${ca_cert_url}"
```

Comment:
it doesn't look like you are plumbing this into the cloud-config-secondary template resource in `config.tfL79-93`

Reply:
resolved in 8059c9d